ingestion/src/metadata/utils/credentials.py

```diff
@@ -76,7 +76,7 @@ def build_google_credentials_dict(gcs_values: GCSValues) -> Dict[str, str]:
     :param gcs_values: GCS credentials
     :return: Dictionary with credentials
     """
-    private_key_str = gcs_values.privateKey.get_secret_value()
+    private_key_str = gcs_values.privateKey.get_secret_value().replace("\\n","\n")
```

Comment:
had to add the replace
as the private key if loaded via env, isn't parsed properly in yaml, 
as the newline character starts the new line from begining when it's expected to start from the indented line.

it doesn't affect the current workflow, and validates the private key for both the approaches